pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp-optimizer-guide/index.md

```diff
@@ -0,0 +1,56 @@
+---
+$title: Using an AMP Optimizer
+$order: 0
+$hidden: true
+description: 'AMP Optimizers are tools that bring AMP Cache optimizations to your own site. Using an AMP Optimizer is key to creating a great page experience and achieving Core Web Vitals compliance. This guides explains how to best use an AMP Optimizer to optimizer your AMP pages.'
+formats:
+  - websites
+  - stories
+author: sebastianbenz
+---
+
+AMP Optimizers are tools that bring AMP Cache optimizations to your own site. Using an AMP Optimizer is key to creating a great [page experience](https://developers.google.com/search/docs/guides/page-experience) and achieving [Core Web Vitals](https://web.dev/vitals/) compliance. If you want to learn more about how an AMP Optimizer works, checkout our [detailed AMP Optimizations Guide](explainer.md).
+
+## Isn’t AMP already fast?
+
+In the beginning, AMP caches served the majority of AMP pages. These caches performed additional optimizations on pages to guarantee a strong user experience. But, over time, more surfaces started linking to AMP pages and developers began building entire websites with AMP.
+
+To ensure best performance and user experience for AMP pages served on the origin. the AMP team
```

Comment:
two nits
1: I believe it should be "To ensure the best performance"
2. Is the second sentence a valediction or just cut off?

Reply:
No idea what I was thinking there. Definitely not intended...